doc/s3.md

```diff
@@ -1,26 +1,40 @@
 # TL;DR
 
-Create and describe s3 buckets
+Create, describe and interact with s3 buckets
 
 ## Use
 
 ### list or ls
+Lists objects in bucketPath if provided, otherwise lists available buckets
 ```
   gen3 s3 list [bucketPath]
 ```
-Lists objects in bucketPath if provided, otherwise lists available buckets
-  - [bucketPath] - s3 path to list
+Options:
+  - [bucketPath]: s3 path to list (e.g. mybucket/subfolder/)
 
-### new 
+### create 
```

Comment:
We'll let this slide, since `gen3 s3` is pretty new, but in general don't change the interface to something people might use in scripts - or at least keep the old way there for a transition period ...